src/Mod/TechDraw/Gui/TaskProjGroup.cpp

```diff
@@ -575,11 +575,20 @@ void TaskProjGroup::spacingChanged()
     if (blockUpdate || !multiView) {
         return;
     }
+
     multiView->spacingX.setValue(ui->sbXSpacing->value().getValue());
     multiView->spacingY.setValue(ui->sbYSpacing->value().getValue());
-    multiView->recomputeFeature();
+
+    // Update view position without recomputing avoiding calling OCC
+    for (App::DocumentObject* obj : multiView->Views.getValues()) {
+        auto* viewItem = dynamic_cast<TechDraw::DrawProjGroupItem*>(obj);
+        if (viewItem) {
+            viewItem->autoPosition();
+        }
+    }
```

Comment:
This code should be simplified. There is a function : 
DrawProjGroup::autoPositionChildren()

So you can just do 
            multiView->autoPositionChildren();